remo/events/templates/edit_event.html

```diff
@@ -338,6 +338,19 @@
 
         <!-- Categories/Goals Section -->
         <div class="row">
+          <!-- Campaign/Intiatives -->
```

Comment:
The UI here seems a bit off, but this PR will be merged after #946 so everything will fall into place.